app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
```

Comment:
Nice. Looks good. Can lines 2, 3, and 5 be consolidated into a one liner?
`validates :username, :email, :phone, uniqueness: true, presence: true`

Reply:
You bet! Good catch!